op-node/rollup/derive/channel_compressor_test.go

```diff
@@ -0,0 +1,60 @@
+package derive
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var r = rand.New(rand.NewSource(99))
+
+func randomBytes(length int) []byte {
+	b := make([]byte, length)
+	_, err := r.Read(b)
+	// Rand.Read always returns nil error
+	if err != nil {
+		panic(err)
+	}
+	return b
+}
+
+func TestSpanChannelCompressor(t *testing.T) {
+	testCases := []struct {
+		name                   string
+		algo                   CompressionAlgo
+		expectedCompressedSize int
```

Comment:
I find the name of this parameter misleading, isn't it more something like the `expectedResetSize`?